server/src/config/index.ts

```diff
@@ -1,18 +1,18 @@
-import { IConfig, LoggerPresets } from './config.types';
+import { IConfig } from './config.types';
 import { EnvHelper } from './helpers/env';
 import { default as loggerPresets } from './presets/logger';
 
 export default {
 	state: {
 		env: process.env.NODE_ENV,
-		docs: EnvHelper.envToBoolean(process.env.STATE_DOCS),
-		production: EnvHelper.envToBoolean(process.env.STATE_PRODUCTION),
-		test: EnvHelper.envToBoolean(process.env.STATE_TEST),
+		docs: EnvHelper.envToBoolean('STATE_DOCS'),
+		production: EnvHelper.envToBoolean('STATE_PRODUCTION'),
+		test: EnvHelper.envToBoolean('STATE_TEST'),
 	},
 	server: {
-		host: process.env.HOST,
-		port: EnvHelper.envToNumber(process.env.PORT),
+		host: 'HOST',
```

Comment:
Dit gaat niet werken, toch? 🤔

Reply:
good catch
fixed

@nielsbril Mss moet ik die ook nog wrappen in een EnvHelper.checkEmpty?

Reply:
@bertyhell Ja, ik was ook zoiets aan het denken. Maar ik wil zelf eens zien: er zit ook Joi validatie op die environment variables, dus dat zit nu eigenlijk op twee plaatsen. Zou eens willen zien of we dat niet op één plaats kunnen krijgen.